generator/views/datagroups.py

```diff
@@ -153,14 +186,27 @@ def generate_datagroups(
         DEFAULT_GENERATED_SQL_URI
     )
 
-    datagroups = sorted(
-        set(
-            datagroup
-            for view in views
-            if (datagroup := _generate_view_datagroup(view, client, dataset_view_map))
-            is not None
-        )
-    )
+    datagroups = set()
+    for view in views:
+        try:
+            if (
+                datagroup := _generate_view_datagroup(view, dataset_view_map, dryrun)
+            ) is not None:
+                datagroups.add(datagroup)
+        except DryRunError as e:
+            if e.error == Errors.PERMISSION_DENIED and e.use_cloud_function:
+                path = (
+                    datagroups_folder_path / f"{e.table_id}_last_updated.datagroup.lkml"
+                )
+                print(
+                    f"Permission error dry running: {path}. Copy existing file from looker-hub."
```

Comment:
This message doesn't quite make sense to me.  Maybe something like this?
```suggestion
                    f"Permission error dry running {view.name}. Copy existing {path} file from looker-hub."
```